docs/doxygen/mainpages/platdetails.h

```diff
@@ -132,12 +132,7 @@ In such case (or when you want to e.g. write a port-specific patch) it can be
 necessary to use the underlying toolkit API directly:
 
 - wxMSW port uses Win32 API: see MSDN docs at https://docs.microsoft.com/en-us/windows/win32/controls/window-controls
-- wxGTK port uses GTK and other lower-level libraries; see
-  - GTK docs at https://developer.gnome.org/gtk/
-  - GDK docs at https://library.gnome.org/devel/gdk/
-  - GLib docs at https://library.gnome.org/devel/glib/
-  - GObject docs at https://library.gnome.org/devel/gobject/
-  - Pango docs at https://library.gnome.org/devel/pango/
-- wxOSX port uses the Cocoa API: see Cocoa docs at https://developer.apple.com/cocoa
+- wxGTK port uses GTK and other lower-level libraries; see https://docs.gtk.org/
+- wxOSX port uses the Cocoa API: see Cocoa docs at https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CocoaFundamentals/WhatIsCocoa/WhatIsCocoa.html
```

Comment:
Not sure if it's worth linking to "archive" docs. Maybe just link to https://developer.apple.com/ instead?

Reply:
I was wondering about that too. The archive page is a fantastic reference for Cocoa, with all the overviews easy to find. The main developer page is quite a bit more generic and you really have to spend some time exploring to find stuff on something like `NSTextView`. And it's very stingy with example code (IMO).

But I'm OK with either option.

Reply:
OK, let's keep the archive link for as long as it still works, thanks.